docs/site/Extending-LoopBack-4.md

```diff
@@ -87,11 +87,8 @@ export class UserManagementComponent implements Component {
   providers?: ProviderMap;
 
   constructor() {
-    this.controllers = {
-      [UserBindings.CONTROLLER]: UserController,
-    };
-    this.repositories = {
-      [UserBindings.REPOSITORY]: UserRepository,
+    this.controllers = [UserController];
+    this.repositories = [UserRepository];
```

Comment:
IIRC, both `controllers` and `repositories` should be objects (key-value maps), not arrays. Did you perhaps mean the following?

```ts
this.controllers = {UserController};
this.repositories = {UserRepository};
```

Reply:
As per the Component interface, `controllers` is an Array of Controller Classes. https://github.com/strongloop/loopback-next/blob/master/packages/core/src/component.ts#L25

Same goes for repositories. 

`providers` is an object.